whois-api/src/main/java/net/ripe/db/whois/api/rdap/RdapObjectMapper.java

```diff
@@ -656,38 +658,27 @@ private static AsBlockRange getAsBlockRange(final String asBlock) {
         }
     }
 
-    private static void handleLanguageAttribute(final RpslObject rpslObject, final RdapObject rdapObject) {
-        final List<RpslAttribute> languages = rpslObject.findAttributes(AttributeType.LANGUAGE);
-        if (languages.isEmpty()) {
-            return;
-        }
-        // TODO: [ES] add warning notice if additional languages found
-        rdapObject.setLang(languages.get(0).getCleanValue().toString());
-        addNoticeForMultipleValues(rdapObject, AttributeType.LANGUAGE, languages, rpslObject.getKey().toString());
-    }
-
-    private static void handleCountryAttribute(final RpslObject rpslObject, final Ip ip) {
-        final List<RpslAttribute> countries = rpslObject.findAttributes(AttributeType.COUNTRY);
-        if (countries.isEmpty()) {
-            return;
-        }
-        // TODO: [ES] add warning notice if additional languages found
-        ip.setCountry(countries.get(0).getCleanValue().toString());
-        addNoticeForMultipleValues(ip, AttributeType.COUNTRY, countries, ip.getHandle());
-    }
-
-    private static void addNoticeForMultipleValues(final RdapObject rdapObject, final AttributeType type, final List<RpslAttribute> values, final String key) {
-        if (values.isEmpty() || values.size() == 1) {
-            return;
-        }
-
-        final String commaSeperatedValues = values.stream().map(RpslAttribute::getCleanValue).collect(Collectors.joining(", "));
-        final String title = String.format("Multiple %s attributes found", type.getName());
-        final String desc = String.format("There are multiple %s attributes %s in %s, but only the first %s %s was returned.", type.getName(), commaSeperatedValues, key, type.getName(), values.get(0).getCleanValue());
-
-        final Notice notice = new Notice();
-        notice.setTitle(title);
-        notice.getDescription().add(desc);
-        rdapObject.getNotices().add(notice);
+    private static void handlePartialValueAttributes(final RpslObject rpslObject, final RdapObject rdapObject){
```

Comment:
The warnings on multiple language or country attributes have been moved into the redacted code, but those warnings are not related to redaction. Can they be kept with the attribute parsing and the redaction kept separate?

Reply:
Agreed that warning notices are replaced by redaction element